api/getCommunalPropertyRepairs/function.json

```
@@ -0,0 +1,17 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": ["get"],
+      "route": "getCommunalPropertyRepairs"
```

The route should not have an API verb prefixed the name. Remove the API verb from the gateway name, i.e. 
```suggestion
      "route": "communalPropertyRepairs"
```